CHANGELOG.md

```diff
@@ -0,0 +1,12 @@
+# Changelog
+
+## master / unreleased
+
+## 0.1.1 / 2020-08-27
+
+* [CHANGE] Use a single producer instead of one per batch. #18
+* [ENHANCEMENT] Cleanup resources properly after SIGTERM.  #18
+
+## 0.1.0 / 2020-08-08
+
+* [FEATURE] Initial release of a Prometheus remote_write adapter for [Pulsar], based on
```

Comment:
This line seems to end prematurely.

Reply:
Good spot, just correcting this in a follow up